### ISSUE TITLE
Add possibility to change author in interactive rebase

### ISSUE THREAD

**Issue by pascalberger** [ForkIssues/TrackerWin#35]
It would be nice to have the possibility to change the author of a commit (`git commit --amend --author="Author Name <email@address.com>"`) in the interactive rebase dialog. While the edit option is there it only seems to allow editing the commit message.

**Comment by Stanzilla** [comment]
Indeed, just had to do this a few times and already wondered why Fork does not support this!
